jina/clients/request/helper.py

```diff
@@ -46,6 +47,11 @@ def _new_request_from_batch(_kwargs, batch, data_type, mode, queryset):
             _add_docs_groundtruths(req, batch, data_type, _kwargs)
         elif mode == RequestType.DELETE:
             _add_ids(req, batch)
+        # TODO make Dump a control request to be passed to the Pod directly
+        elif mode == RequestType.DUMP:
+            req.path = batch
+            req.shards = kwargs.get('shards')
```

Comment:
I would prefer to pass these kwargs specifically, do not like to extract things from `kwargs`. Otherwise maybe we may need to change the Request Constructor?

Reply:
I addressed this in the description and [here](https://github.com/jina-ai/jina/pull/2260#discussion_r604900705)

It's done like this until we have an established refactored way of sending control requests.